cmd/frontend/webhooks/bitbucketserver_webhooks.go

```diff
@@ -0,0 +1,68 @@
+package webhooks
+
+import (
+	"io"
+	"net/http"
+
+	gh "github.com/google/go-github/v43/github"
+
+	"github.com/sourcegraph/log"
+
+	"github.com/sourcegraph/sourcegraph/internal/actor"
+	"github.com/sourcegraph/sourcegraph/internal/extsvc"
+	"github.com/sourcegraph/sourcegraph/internal/extsvc/bitbucketserver"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+func (h *WebhookRouter) HandleBitBucketServerWebhook(logger log.Logger, w http.ResponseWriter, r *http.Request, codeHostURN extsvc.CodeHostBaseURL, payload []byte) {
```

Comment:
I'd rename the receiver to `r` for `router` :)
```suggestion
func (r *WebhookRouter) HandleBitBucketServerWebhook(logger log.Logger, w http.ResponseWriter, r *http.Request, codeHostURN extsvc.CodeHostBaseURL, payload []byte) {
```